src/tutorials/server/httpserver.md

```diff
@@ -7,23 +7,33 @@ prevpage:
   title: Fetch data from the internet
 ---
 
-Here are some resources for writing servers using Dart:
-
-* Documentation
-  * [Using Google Cloud][] has information on Google Cloud products
-    that Dart servers can use, such as Cloud Run.
-  * [Using Google APIs][] points to resources to help you
-    use Firebase and Google client APIs from a Dart app.
-* Samples
-  * [A simple Dart HTTP server][simple-sample]
-    * Uses the [`shelf`][] package.
-    * Also uses the [`shelf_router`][] and [`shelf_static`][] packages.
-    * Is deployable on Cloud Run.
-  * [A Dart HTTP server that uses Cloud Firestore][cloud-sample]
-    * Uses the Cloud Firestore features in the [`googleapis`][] package.
-    * Also uses the [`googleapis_auth`][], [`shelf`][], and
-      [`shelf_router`][] packages.
-    * Is deployable on Cloud Run.
+Dart resources for writing HTTP servers include:
+
+## Packages
```

Comment:
These packages shouldn't go here because they are packages for writing **clients**.

Maybe these should go somewhere on https://dart.dev/tutorials/server/fetch-data